src/app/codelabs/react/introduction/exercise/test.js

```diff
@@ -0,0 +1,81 @@
+import {
+  babylon,
+  babel_traverse as traverse,
+  babel_types as T
+} from '../code';
+
+export default function test(sourceCode) {
+  const tests = [
+    {
+      instruction: `Create a function named App`,
+      condition: ({ node }) => {
+        const isFunction = T.isFunctionDeclaration(node) && node.id.name === 'App';
+        const isVariable = T.isVariableDeclarator(node) && node.id.name === 'App';
+
+        return isFunction || (isVariable && T.isFunctionExpression(node.init)) || (isVariable && T.ArrowFunctionExpression(node.init))
+      }
+    },
+    {
+      instruction: `Return div with some text using JSX`,
+      condition: ({ node }) => {
+        const isDiv = (element) =>
+          T.isJSXElement(element)
```

Comment:
Are you missing a logical operator here? It looks like you meant to include `&&`